libsawtooth/Cargo.toml

```diff
@@ -25,17 +25,24 @@ description = """\
 
 [dependencies]
 cbor-codec = { version = "0.7", optional = true }
+cylinder = "0.1"
 hex = "0.4"
 log = "0.4"
 lmdb-zero = { version = "0.4", optional = true }
 metrics = { version = "0.12", features = ["std"], optional = true }
 openssl = { version = "0.10", optional = true }
 protobuf = "2"
 redis = { version = "0.13.0", default-features = false, optional = true }
-sha2 = "0.8"
-transact = "0.2"
+transact = "0.3"
 uluru = { version = "0.4", optional = true }
 
+[dev-dependencies]
```

Comment:
Looks like you re-ordered some stuff in this commit that's unrelated.

Reply:
Good catch, meant to put that in another commit but forgot it was in here. Fixed.